vignettes/intro-to-hector.Rmd

```diff
@@ -0,0 +1,194 @@
+---
+title: A tour of the Hector R interface
+author: Alexey Shiklomanov
+---
+
+This vignette provides a basic introduction for running Hector with R.
+First, it shows how to do a simple Hector run with one of Hector's built-in scenarios.
+Then, it shows how to modify Hector parameters from within R and uses this functionality to perform a simple sensitivity analysis of several Hector variables to the CO$_2$ fertilization parameter $\beta$.
+
+# Basic run
+
+First, load the hector package.
+
+```{r  load-packages}
+library(hector) 
+```
+
+Hector is configured via an INI file, which defines run metadata, inputs (e.g. emissions scenarios), and parameters.
+For details on this file, see [InputFiles](https://github.com/JGCRI/hector/wiki/InputFiles) on the Hector GitHub Wiki.
+
+These files ship with the Hector R package, in the `input` subdirectory, which allows them to be accessed via `system.file`.
+Below, we get the path of the input file corresponding to the scenario RCP 4.5.
+
+```{r  ini-file }
+ini_file <- system.file("input/hector_rcp45.ini", package = "hector") 
+```
+
+Next, we initialize a Hector instance, or "core", using this configuration.
+This core is a self-contained object that contains information about all of Hector's inputs and outputs.
+The core is initialized via the `newcore` function:
+
+```{r  init-core}
+core <- newcore(ini_file)
+core 
+```
+
+Now that we have configured a Hector core, we can run it with the `run` function.
+
+```{r  run-core }
+run(core) 
+```
+
+Notice that this itself returns no output.
+Instead, the output is stored inside the core object.
+To retrieve it, we use the `fetchvars` function.
+Below, we also specify that we want to retrieve results for the years 2000 to 2300.
+
+```{r  get-results }
+results <- fetchvars(core, 2000:2300)
+head(results) 
+```
+
+The results are returned as a long, tidy `data.frame`.
```

Comment:
Many people won't understand "tidy"; I would make this a link to https://cran.r-project.org/web/packages/tidyr/vignettes/tidy-data.html